features/step_definitions/snippet_steps.rb

```diff
@@ -1,5 +1,6 @@
 Given(/^I am authenticated$/) do
-  user = User.new email: "landry.soules@gmail.com", password: "1Password", username: "landry"
+  # FIXME: is it possible to use factorygirl here (build(:user)) in place of User.new ?
+  user = User.new email: "landry.soules@gmail.com", password: "7ninahartley7", username: "landry"
```

Comment:
Yes, it is even prefered :)

I think you can write `FactoryGirl.create(:user)`